src/plugins/data/public/ui/query_string_input/query_bar_top_row.tsx

```diff
@@ -56,7 +56,7 @@ interface Props {
   indexPatterns?: Array<IIndexPattern | string>;
   intl: InjectedIntl;
   isLoading?: boolean;
-  prepend?: React.ReactNode;
+  prepend?: React.ComponentProps<typeof QueryStringInput>['prepend'];
```

Comment:
I think that there's no reason to take the type from `QueryStringInput`.
It should be the same as the other one - `React.ComponentProps<typeof EuiFieldText>['prepend']`

Reply:
Done!